src/GafferUI/StandardStyle.cpp

```diff
@@ -957,6 +974,12 @@ void StandardStyle::renderLine( const IECore::LineSegment3f &line ) const
 	glCallList( connectionDisplayList() );
 }
 
+void StandardStyle::renderSolidCircle( float radius ) const
+{
+	// \todo: make this a circle.
```

Comment:
Take a look at how the DotNodeGadget tricks the system into giving it a circle, or how [BoxIO does something similar](https://github.com/GafferHQ/gaffer/blob/master/python/GafferUI/BoxIOUI.py#L54).